src/TemplateData.js

```diff
@@ -265,21 +265,27 @@ class TemplateData {
     let dataFileConflicts = {};
 
     for (let j = 0, k = files.length; j < k; j++) {
-      let objectPathTarget = await this.getObjectPathForDataFile(files[j]);
       let data = await this.getDataValue(files[j], rawImports);
+      let objectPathTarget = this.getObjectPathForDataFile(files[j]);
```

Comment:
I removed the await here, since getObjectPathForDataFile is returning a String[] anyways, so there is no Promise to await.